src/targets/gpu/CMakeLists.txt

```diff
@@ -284,7 +284,7 @@ else()
 endif()
 
 if(HAS_ROCBLAS_BETA_FEATURES_API)
-    target_compile_definitions(migraphx_gpu PUBLIC -DROCBLAS_BETA_FEATURES_API -DROCBLAS_NO_DEPRECATED_WARNINGS)
+    target_compile_definitions(migraphx_gpu PUBLIC -DMIGRAPHX_USE_ROCBLAS_TUNING_API -DROCBLAS_BETA_FEATURES_API -DROCBLAS_NO_DEPRECATED_WARNINGS)
```

Comment:
Should DROCBLAS_BETA_FEATURES_API be removed from this list now?  Looks like it's been eliminated?

Reply:
No its still needed to use the API from rocblas.